src/components/ui/SuggestItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classnames from 'classnames';
+import { exact, string, func, bool } from 'prop-types';
+
+const SuggestItem = ({
+  className = '',
+  suggest,
+  isHighlighted = false,
+  onClick,
+  ...rest
+}) =>
+  <>
+    {suggest.categoryLabel &&
+      <h3 className="autocomplete_suggestion__category_title">
```

Comment:
Rendering that here implies we'll create a `<ul>` with some `<h3>` children, which is invalid HTML.
Maybe a better architecture would be to inject a special object at the right place in the data array (before favs or categories), and render them as real list items with no interaction. The only difficulty would be how it affects indices in keyboard nav, but just 'jumping' these items should be easy enough.